### DISCUSSION TITLE
Error in nginx.httppasswd detected

### DISCUSSION THREAD

**Discussion by Ailuj27** [eclipse-ditto/ditto#2052] [General]
Hi all, I have detected an error in the spelling of the **nginx.httpasswd** file. I was trying to deploy ditto using docker, but I could not until I realized that that file is written as **nginx.htpasswd**. The file is located at deployment/docker/nginx.httpasswd. Please, correct the error when you can. 
Thank you.

**Comment by thjaeckle** [comment]
Thanks for the hint - I pushed a fixing commit .. 👍
